### ISSUE TITLE
Fail on wrong integer setting definition in JSON

### ISSUE THREAD

**Issue by nuest** [52North/iceland#24]
When I configure an integer setting as below, there is no error but the setting is initialized with the value ``0``.

```json
    "port" : {
      "type" : "integer",
      "value" : "1234"
    },
```

The correct definition of the integer is of course

```json
    "port" : {
      "type" : "integer",
      "value" : 1234
    },
```

**Comment by autermann** [comment]
[Here](https://github.com/52North/iceland/blob/develop/src/main/java/org/n52/iceland/config/json/JsonSettingsDao.java#L125) could be a check for `node.isNumber()` that throws an exception if it returns `false`. 
Alternatively we could return `node.asInt()` which will try to parse the string, but will also `0` if it fails...

**Comment by nuest** [comment]
Or how about parsing it, and if _that_ fails then an exception?

What should the exception be? Is there a https://github.com/52North/iceland/blob/develop/src/main/java/org/n52/iceland/exception/ConfigurationError.java ?

Regarding the other cases: so we need unit tests I suppose.

**Comment by autermann** [comment]
Fixed in #30.
